Maestro/subscriptions.json

```diff
@@ -509,7 +509,10 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/arcade/blob/master/**/*",
-        "https://github.com/dotnet/machinelearning/blob/master/**/*"
+        "https://github.com/dotnet/machinelearning/blob/master/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/master/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/rel/6.0/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/rel/6.1/**/*",
```

Comment:
Why not `https://github.com/dotnet/uwp-setup/blob/rel/**/*`?

Reply:
Good point, changed.